tests/biomaj_tests.py

```diff
@@ -1,9 +1,12 @@
+from __future__ import print_function
```

Comment:
I am not fond of adding __future__ only to support a print in a test.
Biomaj will go to python3 only in future, python2 support should be best effort.
Python2 print should not need a future import. And for your test case, a logging.error would be best to track log among other ones

Reply:
OK see 577d93d (`logging` was not imported in `biomaj_core` test so we did the same here). We have used `logging.info` because it's not really an error.